.golangci.yml

```diff
@@ -1,51 +1,113 @@
-linters:
+run:
   tests: true
+  timeout: 10m
+
+linters:
   disable-all: true
   enable:
-    - bodyclose
-    - depguard
-    - dogsled
-    - dupl
+    - exportloopref
     - errcheck
+    - gci
     - goconst
     - gocritic
     - gofumpt
     - gosec
     - gosimple
     - govet
     - ineffassign
-    - lll
     - misspell
     - nakedret
-    - prealloc
     - staticcheck
+    - thelper
+    - typecheck
     - stylecheck
+    - revive
     - typecheck
+    - tenv
     - unconvert
+    # Prefer unparam over revive's unused param. It is more thorough in its checking.
     - unparam
     - unused
-    - nolintlint
+    - misspell
 
 issues:
+  exclude-rules:
+    - text: 'differs only by capitalization to method'
+      linters:
+        - revive
+    - text: 'Use of weak random number generator'
+      linters:
+        - gosec
+    - linters:
+        - staticcheck
+      text: "SA1019:" # silence errors on usage of deprecated funcs
+
+  max-issues-per-linter: 10000
+  max-same-issues: 10000
+
 linters-settings:
-  errcheck:
-    check-blank: true
-  depguard:
```

Comment:
do you mind bringing this back?

Reply:
Few minutes.

Reply:
generally, depguard isn't in my lint religion -- did you want it back & configured?